src/neo/Wallets/Wallet.cs

```diff
@@ -347,79 +346,12 @@ private Transaction MakeTransaction(DataCache snapshot, byte[] script, Signer[]
                     tx.SystemFee = engine.GasConsumed;
                 }
 
-                tx.NetworkFee = CalculateNetworkFee(snapshot, tx);
+                tx.NetworkFee = CalculateNetworkFee(snapshot, tx, ProtocolSettings, (a) => GetAccount(a)?.Contract?.Script);
```

Comment:
It seems we still need to rely on `Wallet` for func CalculateNetworkFee

Reply:
You can use this method if you have a dictionary